install.sh

```diff
@@ -102,66 +106,50 @@ function install_local_libs {
 	cp -rf $cachedir/orctlibs/local/* ./lib/.
 }
 
+function install_mingw_32 {
```

Comment:
This is only for OSX for now, so this function should either handle all supported cases or has a name that reflects its true nature.